### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by andriy-shymkiv**
[DEMO LINK](https://andriy-shymkiv.github.io/react_counter/)

### CODE REVIEW

**Review by Artem66 — APPROVED** [review]

File: src/App.tsx [hunk 1]
```diff
@@ -6,19 +6,22 @@ type State = {
 };
 
 export class App extends React.Component<{}, State> {
-  state: Readonly<State> = {
```
Comment (denys-danyliuk) [inline src/App.tsx:9]: why did you delete type?🥲

Comment (Andriy-Shymkiv) [reply src/App.tsx:9]: misclick
